brother_printer_fwupd/__main__.py

```diff
@@ -71,7 +83,7 @@ def parse_args():
         "--fw-versions",
         dest="fw_versions",
         nargs="*",
-        default=list[FWInfo](),
+        default=list(),  # In Python 3.10+: list[FWInfo]
```

Comment:
Please consider using this syntax for lists:

```suggestion
        default=[]
```